### PR TITLE
feat: store created variables in memory

### PR DESCRIPTION
**Pull request by nsalamad**
# Changes
* client now stores variables in memory to be accessed later
* added unit tests
* fixed camel case in `test_config`

### CODE REVIEW

File: DevCycle/Models/UserConfig.swift [hunk 1]
```diff
@@ -147,7 +147,7 @@ public struct Feature {
     }
 }
 
-public struct Variable {
+public class Variable {
```
Comment (jsalaber) [inline DevCycle/Models/UserConfig.swift:150]: any reason to change this from a struct to class?

**Review by jsalaber — CHANGES_REQUESTED** [review]

File: DevCycle/Models/UserConfig.swift (hunk 1 above)
Comment (nsalamad) [reply DevCycle/Models/UserConfig.swift:150]: yeah, I should've mentioned this. The NSMapTable doesn't work with `Any` type, it works with `AnyObject` which doesn't include Structs

Comment (jsalaber) [reply DevCycle/Models/UserConfig.swift:150]: When do you ever need to store this `struct` locally, I thought we only need to store the `DVCVariable` class?

**Review by jsalaber — CHANGES_REQUESTED** [review]

File: DevCycle/Models/UserConfig.swift (hunk 1 above)
Comment (nsalamad) [reply DevCycle/Models/UserConfig.swift]: I think I was doing it incorrectly originally. I changed it back.

**Review by jsalaber — APPROVED** [review]

**Pull request merged**